stock_request/models/stock_request.py

```diff
@@ -128,7 +128,9 @@ def _get_expected_date():
     @api.depends("allocation_ids", "allocation_ids.stock_move_id")
     def _compute_move_ids(self):
         for request in self:
-            request.move_ids = request.allocation_ids.mapped("stock_move_id")
+            move_ids = request.allocation_ids.mapped("stock_move_id")
+            all_moves = move_ids | move_ids.mapped("move_orig_ids")
```

Comment:
Hi @rousseldenis,

We have included the origin moves as well because when there is a route configuration with "Trigger another rule" rules, only the last transfer is linked. 

This is confusing since the stock request actually generates a flow of 2 or 3 transfers that should be linked.

Reply:
Yes but in this case you assume there is only two step route